src/libs/local_store.js

```diff
@@ -1,9 +1,11 @@
 function LocalStore() {}
 
-LocalStore.prototype.getAll = async function () {
+LocalStore.prototype.getAllPois = async function () {
```

Comment:
i'm not sure but when you declare a function async it already return promise. i think you can simplify by just Promise.resolve(JSON.parse...)

Reply:
The async declaration is not from my changes but was already here ?!

Reply:
i say that i think that if you declare method with async, you dont have to return a new Promise